core/controlplane/config/templates/cluster.yaml

```diff
@@ -174,8 +174,29 @@ worker:
 #          # Must be within the range between 100 and 2000
 #          # Defaults to worker.spotFleet.unitRootVolumeIOPS * weightedCapacity if omitted
 #          #rootVolumeIOPS:
+#
+#        #  # Additional EBS volumes mounted on the worker
+#        #  # No additional EBS volume by default. All parameter values do not default - they must be explicitly defined
+#        #  ebsVolumes:
+#        #  - type: "gp2"
+#        #    iops: 0
+#        #    sizeGb: 30
```

Comment:
Would you mind changing this to just `size` so that we can be consistent among other keys like `workerRootVolume"Size"` and `controllerRootVolume"Size"`?